src/include/cpu/rpp_cpu_simd.hpp

```diff
@@ -1072,10 +1180,120 @@ inline void rpp_store48_f32pln3_to_f16pkd3_avx(Rpp16f *dstPtr, __m256 *p)
+    out[2] = _mm512_cvtepu8_epi32 (input[2]);
+    out[3] = _mm512_cvtepu8_epi32 (input[3]);
+
+    __m512i output[3];
+    output[0] = _mm512_mask_blend_epi32 (k, out[0], out[1]);
+    output[1] = _mm512_mask_blend_epi32 (k, out[2], out[3]);
+    output[2] = _mm512_mask_blend_epi32 (k, output[0], output[1]);
+
+    p[0] = _mm512_cvtepu32_ps(output[0]);
+
+    input[0] = _mm512_extracti32x4_epi32(px[1], 0);
+    input[1] = _mm512_extracti32x4_epi32(px[1], 1);
+    input[2] = _mm512_extracti32x4_epi32(px[1], 2);
+    input[3] = _mm512_extracti32x4_epi32(px[1], 3);
+
+    out[0] = _mm512_cvtepu8_epi32 (input[0]);
+    out[1] = _mm512_cvtepu8_epi32 (input[1]);
+    out[2] = _mm512_cvtepu8_epi32 (input[2]);
+    out[3] = _mm512_cvtepu8_epi32 (input[3]);
+
+    output[0] = _mm512_mask_blend_epi32 (k, out[0], out[1]);
+    output[1] = _mm512_mask_blend_epi32 (k, out[2], out[3]);
+    output[2] = _mm512_mask_blend_epi32 (k, output[0], output[1]);
+
+    p[1] = _mm512_cvtepu32_ps(output[0]);
+
+    input[0] = _mm512_extracti32x4_epi32(px[2], 0);
+    input[1] = _mm512_extracti32x4_epi32(px[2], 1);
+    input[2] = _mm512_extracti32x4_epi32(px[2], 2);
+    input[3] = _mm512_extracti32x4_epi32(px[2], 3);
+
+    out[0] = _mm512_cvtepu8_epi32 (input[0]);
+    out[1] = _mm512_cvtepu8_epi32 (input[1]);
+    out[2] = _mm512_cvtepu8_epi32 (input[2]);
+    out[3] = _mm512_cvtepu8_epi32 (input[3]);
+
+    output[0] = _mm512_mask_blend_epi32 (k, out[0], out[1]);
+    output[1] = _mm512_mask_blend_epi32 (k, out[2], out[3]);
+    output[2] = _mm512_mask_blend_epi32 (k, output[0], output[1]);
+
+    p[2] = _mm512_cvtepu32_ps(output[0]);
+
+    input[0] = _mm512_extracti32x4_epi32(px[3], 0);
+    input[1] = _mm512_extracti32x4_epi32(px[3], 1);
+    input[2] = _mm512_extracti32x4_epi32(px[3], 2);
+    input[3] = _mm512_extracti32x4_epi32(px[3], 3);
+
+    out[0] = _mm512_cvtepu8_epi32 (input[0]);
+    out[1] = _mm512_cvtepu8_epi32 (input[1]);
+    out[2] = _mm512_cvtepu8_epi32 (input[2]);
+    out[3] = _mm512_cvtepu8_epi32 (input[3]);
+
+    output[0] = _mm512_mask_blend_epi32 (k, out[0], out[1]);
+    output[1] = _mm512_mask_blend_epi32 (k, out[2], out[3]);
+    output[2] = _mm512_mask_blend_epi32 (k, output[0], output[1]);
+
+    p[3] = _mm512_cvtepu32_ps(output[0]);
+
+    input[0] = _mm512_extracti32x4_epi32(px[4], 0);
+    input[1] = _mm512_extracti32x4_epi32(px[4], 1);
+    input[2] = _mm512_extracti32x4_epi32(px[4], 2);
+    input[3] = _mm512_extracti32x4_epi32(px[4], 3);
+
+    out[0] = _mm512_cvtepu8_epi32 (input[0]);
+    out[1] = _mm512_cvtepu8_epi32 (input[1]);
+    out[2] = _mm512_cvtepu8_epi32 (input[2]);
+    out[3] = _mm512_cvtepu8_epi32 (input[3]);
+
+    output[0] = _mm512_mask_blend_epi32 (k, out[0], out[1]);
+    output[1] = _mm512_mask_blend_epi32 (k, out[2], out[3]);
+    output[2] = _mm512_mask_blend_epi32 (k, output[0], output[1]);
+
+    p[4] = _mm512_cvtepu32_ps(output[0]);
+
+    input[0] = _mm512_extracti32x4_epi32(px[5], 0);
```

Comment:
Please change the variable names as per the RPP style
variable names should start with px for interger and p for float
Please give better variable names which are meaningful for registers instead of input, in, output, out